src/sage/groups/abelian_gps/abelian_group.py

```diff
@@ -1175,7 +1179,7 @@ def random_element(self):
             order = g.order()
             if order is infinity:
                 order = 42  # infinite order; randomly chosen maximum
-            result *= g**(randint(0,order))
+            result *= g**(randint(0, order))
```

Comment:
```suggestion
            result *= g ** (randint(0, order))
```
If you are going through and making PEP8 changes (although I generally advise against that to avoid merge conflicts and noise in a change; here it is fairly minimal, so IMO it isn't a problem).